isolate_example/README.md

```diff
@@ -1,16 +1,24 @@
 # isolate_example
 
-A Flutter sample to demonstrate isolates
+A sample application that demonstrate best practices when using [`isolates`](https://api.dartlang.org/stable/2.3.1/dart-isolate/Isolate-class.html). in several cases with flutter.
+Flutter SDK to isolates, which include:
 
-## Getting Started
+## Goals
 
-This project is a starting point for a Flutter application.
+* Display the performance benefits of isolates when using them in the right situation 
+* Show how to use the compute method for straightforward computations
```

Comment:
nit: put backticks around `compute` here, since it's a source code reference. That'll get it formatted in the code font.